scala.html.markdown

```diff
@@ -276,6 +276,7 @@ r foreach println
 // NB: Scala is quite lenient when it comes to dots and brackets - study the
 // rules separately. This helps write DSLs and APIs that read like English
 
+// Why doesn't `println` need any parameters here? Stay tuned for Functional Programming below!
```

Comment:
Aha! I see, I misunderstood. Yes, that's a good point. I was referring to the "Functional Programming" section that comes next, but it would be good to point out what it is. Perhaps this?

    // Stay tuned for first-class functions in the Functional Programming section below!